pkg/apimachinery/patch/patch.go

```diff
@@ -119,6 +119,18 @@ func (p *PatchSet) GeneratePayload() ([]byte, error) {
 	return GeneratePatchPayload(p.patches...)
 }
 
+func (p *PatchSet) GenerateStringPatchOperations() ([]string, error) {
```

Comment:
Do you need this? Isn't simple conversion enough?

Reply:
IMO it does not fit as an API addition to PatchSet.
If the logic is unique to some specific part of the code, it should be done there.

I would avoid doing it here to assure not to encourage using it in other places.

Reply:
Agreed! That was one of my doubts actually, will move it. Thanks